### ISSUE TITLE
terminal closes right after launch

### ISSUE THREAD

**Issue by rykelley** [withfig/fig#1931]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

Ubuntu 22.04

### Expected behaviour

For fig to launch and autocomplete

### Actual behaviour

during the fig install, it asks for my GitHub HTTPS credentials. after I run those the terminal crashes and crashes on every launch after that. i have to use an editor to comment out the fig top and bottom lines for my terminal to work again. here is fig doctor after I comment out those lines 

fig doctor

✘ ~/.bashrc contains valid fig hooks: /home/rykelley/.bashrc does not source pre integration first
  Run fig install --dotfil es to reinstall shell integrations for bash
Attempting to fix automatically...
Re-running check...

✔ ~/.bashrc contains valid fig hooks
✘ Fig Integration: Figterm is not running
  FIG_INTEGRATION_VERSION=None

✘ Doctor found errors. Please fix them and try again.

If you are not sure how to fix it, please open an issue with fig issue to let us know!
Or, email us at hello@fig.io!

### Steps to reproduce

Running fig install --dotfiles

### Environment

```yaml
❯ fig diagnostic

fig-details:
  - cli-version: 1.5.0
  - desktop-version: 2.5.3
  - figterm-version: 5.2.0
hardware-info:
  - chip-id: 11th Gen Intel(R) Core(TM) i5-1135G7 @ 2.40GHz
  - cores: 4
  - mem: 63929.504 GB
os-info:
  - kernel: 5.15.0-52-generic
  - distro: "Ubuntu"
  - distro-version: "22.04.1 LTS (Jammy Jellyfish)"
environment:
  - shell: /usr/bin/bash
  - terminal: <unknown>
  - cwd: /home/rykelley
  - exe-path: /usr/bin/fig
  - install-method: unknown
  - env-vars:
    - SHELL: /bin/bash
    - XMODIFIERS: @im=ibus
    - XDG_SESSION_DESKTOP: ubuntu
    - XDG_SESSION_TYPE: wayland
    - XDG_CURRENT_DESKTOP: ubuntu:GNOME
    - TERM: xterm-256color
    - DISPLAY: :0
    - QT_IM_MODULE: ibus
    - PATH: /home/rykelley/gems/bin:/home/rykelley/.local/bin:/home/rykelley/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin
```

**Comment by grant0417** [comment]
Could you run `fig doctor --verbose`

**Comment by rykelley** [comment]
fig doctor --verbose
Let's check if you're logged in...
✔ Logged into Fig

Let's check your dotfiles...
✔ ~/.bashrc contains valid fig hooks
✔ ~/.profile contains valid fig hooks

Let's make sure Fig is setup correctly...
✔ Fig bin exists
✔ PATH contains ~/.local/bin
✘ Fig Integration: Figterm is not running
  FIG_INTEGRATION_VERSION=None

Let's make sure Fig is running...
✔ Daemon
✔ Daemon diagnostics

Let's see if Fig is in a working state...
✘ Figterm: No TERM_SESSION_ID
  
   0: No TERM_SESSION_ID
   1: environment variable not found

Location:
   fig_cli/src/cli/doctor.rs:600

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
✔ Insertion lock does not exist
✔ PATH and PseudoTerminal PATH match
✔ Autocomplete dev mode
✔ Plugin dev mode
✔ Mission Control is loading from the correct URL

Let's check if your system is compatible...
● Fig's support for Linux 5.15.0-52-generic - "Ubuntu" is in development. It may not work properly on your system.


Let's check fig diagnostic...
✘ Autocomplete is active: Autocomplete is currently inactive. Your desktop integration(s) may be broken!

Let's check your terminal integrations...

Let's check Linux integrations
✘ IBus Env Check: IBus environment variable is not set
  Please restart your DE/WM session, for more details see https://fig.io/user-manual/other/linux
  GTK_IM_MODULE is not set, expected 'ibus'
✔ IBus Check
✔ Fig is not running in a sandbox


  Fig still not working? Run fig issue to let us know!
  Or, email us at hello@fig.io!


Strange, i checked the ibus req's before ...

**Comment by rykelley** [comment]
so this is getting stuck on when trying to clone this repo https://github.com/greymd/docker-zsh-completion/  
![image](https://user-images.githubusercontent.com/1408442/207150927-4d552894-aee2-4aa9-bb97-57057568bcd1.png)
![Screenshot from 2022-12-12 15-45-56](https://user-images.githubusercontent.com/1408442/207151004-1707a022-573c-4d07-a589-608680017f6d.png)

**Comment by rykelley** [comment]
yep, i have done that repeatedly hoping for a bug fix, same issue. i also have a git cred manager running so even though it's a public repo it's not a permission thing on my end.

**Comment by rykelley** [comment]
where does that code need to go ?

**Comment by grant0417** [comment]
Its really weird to see your version is 2.5.3, our latest version is 2.9 which went live 2 weeks ago, can you try `sudo apt update && sudo apt upgrade`?
